ui/src/shared/components/TableGraph.js

```diff
@@ -80,25 +110,55 @@ class TableGraph extends Component {
     }
   }
 
+  handleClickFieldName = (columnIndex, rowIndex) => () => {
+    const {tableOptions} = this.props
+    const {clicktoSortFieldIndex, clicktoSortDirection, data} = this.state
```

Comment:
Variable not in camelCase, shouldnt it be 'clickToSort...'?

Reply:
thank you.